src/pages/layer-2.js

```diff
@@ -0,0 +1,629 @@
+          availability layer for Layer 2s, and if there are any disputes on
+          previous transactions data is provided from Ethereum for these
+          disputes.
+        </p>
+        <p>
+          While Ethereum has been a breakthrough technology in smart contract
+          blockchains, it faces the same trilemma between scaling,
+          decentralization, and security as any other blockchain. Ethereum has
+          done a great job addressing decentralization, and security, with plans
+          to address scalability with sharding in the future. While there are
+          roadmap plans to improve the scalability of Ethereum, there is still a
+          need for further scaling to round out the trilemma for Ethereum. This
+          is where Layer 2 technology is being developed to step in and bring
+          further scaling to Ethereum. As time has progressed, Ethereum as a
+          layer 1 is a settlement layer for layer 2s, and not the layer for end
+          users.
+        </p>
+      </Content>
+
+      <Content>
+        <h2>Why do we need layer 2?</h2>
+        <p>
+          The three desirable properties of a blockchain are that it is
+          decentralized, secure, and scalable. The scalability trilemma states
+          that a simple blockchain architecture can only achieve two out of
+          three. Want a secure and scalable blockchain? You need to sacrifice
+          decentralization.
+        </p>
+        <p>
+          Ethereum has reached the networks capacity with{" "}
+          <Link to="https://etherscan.io/chart/tx">
+            1+ million transactions per day
+          </Link>
+          , with high demand for each of these transactions. Due to the success
+          of Ethereum and the demand to use it, gas prices have risen
+          substantially, increasing the{" "}
+          <Link to="/developers/docs/scaling/">need for scaling solutions</Link>
+          . The main goal of scalability is to increase transaction speed
+          (faster finality), and transaction throughput (high transactions per
+          second), without sacrificing decentralization or security (more on the
+          Ethereum vision).
+        </p>
+        <p>
+          The Ethereum community has taken a strong stance that it would not
+          throw out decentralization or security in order to scale. Until
+          sharding, Ethereum Mainnet (layer 1) will only be able to process
+          roughly 15 transactions per second. When demand to use Ethereum is
+          high this causes network congestion, increasing transaction fees, and
+          pricing out those who cannot afford it from using Ethereum until the
+          fees reduce. That is where layer 2 comes in.
+        </p>
+      </Content>
+
+      <Content>
+        <InfoGrid>
+          {layer2Cards.map(({ image, title, preview, body, alt }, idx) => (
+            <ExpandableCard
```

Comment:
I don't really see the benefit of using ExpandableCards here. The extra to expand click seems like unneeded friction. What about something similar to what we did for the [the community page](https://ethereum.org/en/community/)?

<img width="1483" alt="Screenshot 2022-03-14 at 12 33 22" src="https://user-images.githubusercontent.com/62268199/158173171-9d8d6009-a74e-4aba-8e4f-c922414c1152.png">

Reply:
Agreed, will work with Jakub and Nuno on this!